File: DSA/Tanisha_500125283/Day2Question2.java
// Problem Statement: Given an array nums of size n, return the first element and the last element of the array as a pair. 
// Task: Return the first and last elements of the array. 
// Examples Input: nums = [1, 2, 3, 4] 
// Output: First: 1, Last: 4

package DSA.Tanisha_500125283;

public class Day2Question2 {
    public static void firstAndLastElement(int[] num){
        if(num==null || num.length==0){
            System.out.println("Array is empty");
        }
        int first_element=num[0];
        int last_element=num[num.length-1];
        System.out.println("First: "+ first_element+", Last: "+last_element);

    }
    public static void main(String[] args){
        int[] num={1,2,3,4};
        firstAndLastElement(num);
    }
}

File: DSA/ShatakshiMishra_590015008/Day2Question2.cpp
#include <iostream>
#include <vector>
using namespace std;

int main() {
    int n;
    cin >> n;

    vector<int> nums(n);
    for (int i = 0; i < n; i++) {
        cin >> nums[i];
    }

    if (n == 0) {
        cout << "Array is empty" << endl;
    } else {
        cout << "First: " << nums[0] << ", Last: " << nums[n - 1] << endl;
    }
}
